### PR TITLE
Added Abstract API

### PR DESCRIPTION
**Pull request by Emmajperrez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1009,7 +1010,7 @@ Table of Contents
 
 
 ## Screenshot APIs
-
+  * [Abstract Screenshot API](https://www.abstractapi.com/website-screenshot-api) — Screenshot API handling resizing, multiple image formats, css injections and javascript rendering.
```
Comment (ripienaar) [inline README.md:1013]: Thank you, I'd prefer we list it once only

Comment (Emmajperrez) [reply README.md:1013]: Sure!
Do you expect me to remove and resubmit a PR?

**Comment by Emmajperrez** [comment]
I just made a new commit to remove the 2nd mention.
Thank you so much!

**Comment by ripienaar** [comment]
thanks :)

**Pull request merged**